### ISSUE TITLE
Calculate T1 value in cirq.experiments.t1_decay

### ISSUE THREAD

**Issue by asmuzsoy** [quantumlib/Cirq#4264]
**Is your feature request related to a use case or problem? Please describe.**
The [cirq.experiments.t1_decay](https://github.com/quantumlib/Cirq/blob/master/cirq-core/cirq/experiments/t1_decay_experiment.py) function returns a Pandas dataframe of the delay time, false count, and true count. It doesn't return the actual value of the T1 parameter, which is the exponential decay constant of the probability of being in the excited state:

<img src="http://www.sciweavers.org/tex2img.php?eq=P%28%7C1%20%5Crangle%20%29%20%3D%20e%5E%7B-%5Cfrac%7Bt%7D%7BT_1%7D%7D&bc=White&fc=Black&im=jpg&fs=12&ff=txfonts&edit=0" align="center" border="0" alt="P(|1 \rangle ) = e^{-\frac{t}{T_1}}" width="87" height="22" />

(formula from https://quantumcomputing.stackexchange.com/questions/9752/whats-the-difference-between-t1-and-t2)

**Describe the solution you'd like**
Since we have the time and probability points already, we can fit those to an exponential decay function (perhaps using scipy.optimize.curve_fit or something similar) and calculate the T1 constant. This can then be added as a method of the T1DecayResult object.

**[optional] Describe alternatives/workarounds you've considered**
This can be done on an individual basis but it would be nice to have a built-in function for it.

**What is the urgency from your perspective for this issue? Is it blocking important work?**
 
P3 - I'm not really blocked by it, it is an idea I'd like to discuss / suggestion based on principle

**Comment by dabacon** [comment]
This appears to be fixed in #4290
